modules/auth/site/scripts/auth_temp_users.tpl

```diff
@@ -0,0 +1,13 @@
+
+DROP TABLE IF EXISTS "auth_temp_users";
```

Comment:
Are you sure we need this "DROP" line ?

usually we should have an install and uninstall script.